packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -154,20 +189,19 @@ class MultiSelectList extends React.Component {
     return this.props.children.length;
   }
 
-  unusedVariables(variable) {
-    return variable === this;
-  }
-
   render() {
-    const { children, isDivided, onChange, maxSelectionCount, ...customProps } = this.props;
+    const { children, isDivided, ...customProps } = this.props;
```

Comment:
@StephenEsser has this same sort of logic in the responsive element. Instead of removing these params, what do you think about just ignoring the eslint rule?

https://github.com/cerner/terra-core/blob/master/packages/terra-responsive-element/src/ResponsiveElement.jsx#L101-L102